speakers.html

```diff
@@ -0,0 +1,131 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
redundand space

Reply:
ale że między nad head? bo jak robię format document to samo mi tak poprawia odstępy O.o

Reply:
*bez "między"

Reply:
to niech tak zostanie ale nie jest konieczna